base/dcp_dest.go

```diff
@@ -345,6 +345,7 @@ func StartCbgtGocbFeed(bucket Bucket, spec BucketSpec, args sgbucket.FeedArgumen
 	// Close the feed if feed terminator is closed
 	if args.Terminator != nil {
 		go func() {
+			defer close(args.DoneChan)
```

Comment:
When a function is only a few lines, and only has a single exit path, I think it's preferable to simply put this at the end of the function, instead of using a defer.

Reply:
This makes sense, changes are committed.